### PR TITLE
Update ug format

### LINKED ISSUE #158: [UG] Update numbering format at the start of the document
image

### LINKED ISSUE #159: [UG] Standardise spacing between word and colon
Suggest leaving no space, e.g. `Viewing help: help` instead of `Viewing help : help`

### LINKED ISSUE #161: [UG] Use more user-centric language

### LINKED ISSUE #163: [UG] Include legend for symbols used in the document
e.g. for the info icon etc.

### LINKED ISSUE #165: [UG] Order features either alphabetically or by most common usage
- [ ] Mention the ordering that we used somewhere in the document
- [ ] Summary command table at the bottom should follow this ordering.

### LINKED ISSUE #166: [UG] Sort's parameter should be capitalized
Follows from the user guide notes which states `Words in UPPER_CASE are the parameters to be supplied by the user`

image

### PR DESCRIPTION
**Pull request by m0nggh**
Made some readjustments to the UGs as much as possible according to the review comments.
- Update numbering format and standardise spacings
- Made it more user-centric
- Include a simple legend (can update more if necessary)
- Rearranged the features in alphabetical order (please say if another format is preferred)
- Sort's parameter is capitalised to suit the command format

Resolves #158 
Resolves #159 
Resolves #161 
Resolves #163 
Resolves #165 
Resolves #166

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -2,31 +2,46 @@
...
 
 ## Introduction
-ZooKeep is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ZooKeep can get your management tasks done faster than traditional GUI apps.
+`ZooKeep` is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, `ZooKeep` can get your management tasks done faster than traditional GUI apps.
+
+## Legend
+
+<div markdown="block" class="alert alert-info">
+
+**:information_source: Provides additional notes and constraints for certain information.**<br>
```
Comment (aizatazhar) [inline docs/UserGuide.md]: i think you can delete the `<br>` at the end of this line cos it makes the spacing off

File: docs/UserGuide.md [hunk 2]
```diff
@@ -181,39 +214,51 @@ Example:
...
+There is no need to save manually, though you can create a copy of the current data with the `snap` command which will be explained below.
+
+</div>
 
 ## FAQ
 
 **Q**: How do I transfer my data to another Computer?<br>
 **A**: Install the app in the other computer and overwrite the empty data file it creates with the file that contains the data of your previous ZooKeep home folder.
 
+**Q**: Can I record the feeding times of a specific animal in any order I prefer?
```
Comment (aizatazhar) [inline docs/UserGuide.md]: add `<br>` to the end of this line

File: docs/UserGuide.md [hunk 3]
```diff
@@ -181,39 +214,51 @@ Example:
...
+**A**: The feeding time will be arranged in chronological order regardless of the order entered for easier reference.
+
+**Q**: Can I search for animals based on a certain alphabet or half specified keywords?
```
Comment (aizatazhar) [inline docs/UserGuide.md]: add `<br>` to the end of this line

File: docs/UserGuide.md [hunk 4]
```diff
@@ -181,39 +214,51 @@ Example:
...
+**A**: The find feature will only list animals with the exact specified keyword provided.
```
Comment (aizatazhar) [inline docs/UserGuide.md:248]: maybe can include that it is case insensitive?

Comment (m0nggh) [reply docs/UserGuide.md:248]: I think i placed it as a tip under the find feature, thanks anyways!

File: docs/UserGuide.md [hunk 5]
```diff
@@ -2,31 +2,46 @@
...
 
-1. Type the command in the command box and press Enter to execute it, e.g. typing **`help`** and pressing Enter will open the help window.
+1. Type the command you want in the command box and press Enter to execute it, e.g. typing **`help`** and pressing Enter will open the help window.
    <br> Here are some example commands you can try:
 
    * **`list`** : Lists all animals.
    * **`exit`** : Exits the app.
 
-1. Refer to the [Features](#features) below for details of each command.
+1. You can refer to the [Features](#features) below for the details of each command. The features are categorised in alphabetical order for easier reference.
```
Comment (lamlaaaam) [inline docs/UserGuide.md]: Minor nitpick in language. "listed" instead of "categorised".

**Comment by lamlaaaam** [comment]
Just a thought, is listing the features alphabetically the best bet? Currently it references the less "basic" features first like "append". Furthermore these features reference features that are later down the list, meaning the user would not have read them yet. Shall we consider listing from basic to advanced?

File: docs/UserGuide.md [hunk 6]
```diff
@@ -2,31 +2,46 @@
...
+* Table of Contents
 {:toc}
 
 ## Introduction
-ZooKeep is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, ZooKeep can get your management tasks done faster than traditional GUI apps.
+`ZooKeep` is a desktop app for managing animals under a zookeeper’s care, optimized for use via a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, `ZooKeep` can get your management tasks done faster than traditional GUI apps.
+
+## Legend
+
+<div markdown="block" class="alert alert-info">
```
Comment (aizatazhar) [inline docs/UserGuide.md]: sorry i think the reason why the spacing is weird is bc it is a block. could you change it to try changing it to `markdown="span"`?

**Pull request merged**